scopes/lanes/lanes/switch.cmd.ts

```diff
@@ -91,7 +91,7 @@ export class SwitchCmd implements Command {
         )}\n`;
         return `${title} ${applyVersionReport(components, false)}${laneSwitched}`;
       }
-      const title = `successfully switched the following components to the version of ${lane}\n\n`;
+      const title = `successfully switched the following components to the head of lane ${lane}\n\n`;
```

Comment:
ok makes sense, changed to 'local head'

Reply:
but it can be also the remote head :)  
So I suggest either, leaving it with "version" or changing to "head" (which can be interpreted as either local or remote).

Reply:
done